src/open_inwoner/configurations/models.py

```diff
@@ -584,6 +588,81 @@ def get_help_text(self, request) -> Optional[str]:
             return ""
 
 
+class CustomFontField(models.FileField):
+    def __init__(self, file_name: str = "", **kwargs):
+        self.file_name = file_name
+        super().__init__(**kwargs)
+
+
+class CustomFontSet(models.Model):
+    def update_filename(self, filename: str, new_name: str, path: str) -> str:
+        ext = filename.split(".")[1]
+        filename = f"{new_name}.{ext}"
+        return "{path}/{filename}".format(path=path, filename=filename)
+
+    def update_filename_body(self, filename: str) -> str:
+        return CustomFontSet.update_filename(
+            self,
+            filename,
+            new_name=CustomFontName.body,
+            path="custom_fonts/",
+        )
+
+    def update_filename_heading(self, filename: str) -> str:
+        return CustomFontSet.update_filename(
+            self,
+            filename,
+            new_name=CustomFontName.heading,
+            path="custom_fonts/",
+        )
```

Comment:
- Model methods go below the model fields.

- Also this code is really unusual: it calls instance methods with the class and explicit self? 

````python
    # why this?
    return CustomFontSet.update_filename(
            self,
            filename,
            new_name=CustomFontName.heading,
            path="custom_fonts/",
        )
    # and not this like normal?
    return self.update_filename(
            filename,
            new_name=CustomFontName.heading,
            path="custom_fonts/",
        )
````

Reply:
- I can't put the methods below the fields because the method names are referenced in `upload_to`, and I can't use strings referencing the methods (as you would when referencing a model that's defined further down in the code) because Django will interpret the string as the destination of `upload_to`. Is there any other way to reference methods in model fields?
- Class/explicit `self`: confusion on my part

Reply:
Oh, I missed these are the upload_to callback. Usually callbacks are put above the class (they're not really methods but Django just calls them with the instance as first argument so it looks like it).